sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Scenario/ManagedInstanceTests.cs

```diff
@@ -62,7 +62,7 @@ public async Task GlobalSetUp()
                         AddressPrefix = "10.10.2.0/24",
                         Delegations =
                         {
-                            new ServiceDelegation() { ServiceName  = "Microsoft.Sql/managedInstances",Name="Microsoft.Sql/managedInstances" ,ResourceType="Microsoft.Sql"}
+                            new ServiceDelegation() { ServiceName  = "Microsoft.Sql/managedInstances",Name="Microsoft.Sql/managedInstances" ,ResourceType="/Microsoft.Sql"}
```

Comment:
Is the slash used to bypass the `ResourceType` validation? Based on @ArthurMa1978's investigation, can we change this property value to `Microsoft.Sql/managedInstances`?

Reply:
All the above modifications are complete